### PR TITLE
Admin instructions for write to councillors

### PR DESCRIPTION
**Pull request by henare**
This is the start of modifying the readme to make it more useful for admins adding councillors.

### CODE REVIEW

**Comment by henare** [comment]
I think this is ready to go. We still need #969 done before we merge it though.

@equivalentideas I'm assigning it to you to review as you work through adding councillors today.

File: README.md [hunk 1]
```diff
@@ -106,12 +108,16 @@ You can see which authorities have the feature enabled at the Authorities admin
 
 #### Adding councillors for an authority
```
Comment (equivalentideas) [inline README.md]: The current structure of this section makes it a bit hard to follow as a check list, which is how use use it as you work through the steps.

I think front loading the key actions in each par, and even using bold emphasis to highlight the steps could help.

I'll give that a go.

Comment (henare) [inline README.md]: :ok_hand:

**Comment by equivalentideas** [comment]
> I think front loading the key actions in each par, and even using bold emphasis to highlight the steps could help.

I've done this and added a little more explanation where I think it's needed based on my experience of going through the process.

I'll merge this and we can continue to iterate. I wont wait for tests since this only changes the readme.
